app/javascript/react/components/MonsterShow.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+
+
+
+const MonsterShow = (props) => {
```

Comment:
Extra line spaces can be removed, you can keep one for separation

Reply:
Done